### PR TITLE
Add definitions for `definitions.person`

### PR DESCRIPTION
**Pull request by sdruskat**
**Related issues**

Refs: #206

Includes: #256 

**Describe the changes made in this pull request**

- Despite its title, also adds all available fields to the first entity definition example

**Instructions to review the pull request**

- Check if this all makes sense (i.e., look at `schema-guide.md`)

### CODE REVIEW

**Comment by sdruskat** [comment]
Zenodo is currently down, hence the link checker fails.

**Comment by jspaaks** [comment]
I took the liberty of merging 1.2.0 into this for easier diff, hope you dont mind

File: schema-guide.md [hunk 1]
```diff
@@ -1618,6 +1699,16 @@ An entity can represent different types of entities, such as a team, an institut
 - **description**: The person's post code.
 - **usage**:<br><br>
     ```yaml
+    authors:
+      - family-names: Druskat
+        given-names: Stephan
+        post-code: 90210
+    ```
+        ```yaml
```
Comment (jspaaks) [inline schema-guide.md:1707]:
```suggestion
    ```yaml
```

**Review by jspaaks — APPROVED** [review]
Good to go

**Comment by sdruskat** [comment]
> I took the liberty of merging 1.2.0 into this for easier diff, hope you dont mind

Course not, and thanks for the review!

**Pull request merged**